q2_cutadapt/_trim.py

```diff
@@ -75,6 +85,8 @@ def _build_trim_command(f_read, r_read, trimmed_seqs,
         '--times', str(times),
         '--overlap', str(overlap),
         '--minimum-length', str(minimum_length),
+        '-q', ','.join([str(quality_cutoff_3end), str(quality_cutoff_5end)]),
```

Comment:
**Bug:**
I believe this string join is backwards, it should be the 5' (front) first, and then the 3' (rear).

```
  -q [5'CUTOFF,]3'CUTOFF, --quality-cutoff [5'CUTOFF,]3'CUTOFF
                        Trim low-quality bases from 5' and/or 3' ends of each
                        read before adapter removal. Applied to both reads if
                        data is paired. If one value is given, only the 3' end
                        is trimmed. If two comma-separated cutoffs are given,
                        the 5' end is trimmed with the first cutoff, the 3' end
                        with the second.
```

Since there's always a default it in the q2 plugin, it does look like we don't have to worry about the user passing only one or the other and having cutadapt always consider it the 3' end, which is good. Although having some tests which specifically exercise passing only one of the two parameters would be a good way to prevent this issue sneaking up on us in the future. Since anyone working with this section would be forced to consider both options and how that effects the command.